burstphoto/ContentView.swift

```diff
@@ -216,16 +227,32 @@ struct SettingsView: View {
             Spacer()
             
             VStack(alignment: .leading) {
-                Text("Robustness").font(.system(size: 14, weight: .medium))
+                Text("Noise reduction:  \(Int(AppSettings.noise_reduction+0.5))")
+                    .font(.system(size: 14, weight: .medium))
+                    .foregroundColor(isEditing ? .secondary : .primary)
```

Comment:
While being very important to update the website, this work seems to be independent from the app development itself.

Reply:
This comment doesn't have any impact on the code content of the PR. I included as a reminder for Martin to make sure he updates any external documentation since they will be out of date otherwise.